src/app/handlers/blacklist.handler.ts

```diff
@@ -2,37 +2,61 @@ import { TextChannel } from 'discord.js';
 import Constants from '../../common/constants';
 import { IContainer, IHandler, IMessage, ClassType } from '../../common/types';
 import { Moderation } from '../../services/moderation.service';
+import { readFileSync } from 'fs';
+//Import VaderSentiment
+const vader = require('vader-sentiment');
+//Try to open the black listed words file
+let blacklistedWords: string[];
+try {
+  blacklistedWords = (readFileSync('src/common/wordblacklist.txt', 'utf-8')).split(/\r?\n/);
+} catch (error) {
+  blacklistedWords = [];
+  console.error(error);
+}
 
 interface LinkLabel {
   regex: RegExp;
   label: string;
 }
-
+// List of all Blacklisted Links in the server
 export class BlacklistHandler implements IHandler {
   private _expressions: LinkLabel[] = [
     { regex: /discord\.gg/, label: 'discord' },
     { regex: /group\s?me/, label: 'GroupMe' },
     { regex: /chegg\.com/, label: 'Chegg' },
     { regex: /coursehero\.com/, label: 'CourseHero' },
     { regex: /quizlet\.com/, label: 'Quizlet' },
+    { regex: /ucf\.zoom\.us/, label: 'Zoom' },
   ];
 
   private _whitelistedChannels = new Set([Constants.Channels.Public.Clubs]);
   constructor(public container: IContainer) {}
 
   public async execute(message: IMessage): Promise<void> {
     const channel = message.channel as TextChannel;
+    const intensity = vader.SentimentIntensityAnalyzer.polarity_scores(message.content);
+
+    //Word Black List
+    if (blacklistedWords.some((blockedWords) => 
```

Comment:
save the result of the some() call into a variable for more readable `if` conditon